lib/RTCSession.js

```diff
@@ -3523,4 +3534,21 @@ module.exports = class RTCSession extends EventEmitter
       video
     });
   }
+
+  _getDTMFRTPSender()
+  {
+    const streams = this._connection.getLocalStreams();
+
+    if (streams.length > 0)
+    {
+      const audioTracks = streams[0].getAudioTracks();
```

Comment:
Honestly I do not like the idea of assuming that there is just a single audio track. Yup, I know that is common in VoIP and SIP regular calls, but forcing JsSIP to assume that by design is not something that makes me too happy.

@jmillan thoughts?

Reply:
I find it OK sending the DTMFs to the **first audio track found**.

Having said that, we no longer use `getLocalStreams()`. See [this].(https://github.com/versatica/JsSIP/blob/master/lib/RTCSession.js#L3346)

Also `createDTMFSender()` is obsoleted in favour of [RTCDTMFSender](https://www.w3.org/TR/webrtc/#dom-rtcdtmfsender)